perllib/FixMyStreet/Cobrand/Oxfordshire.pm

```diff
@@ -47,5 +47,14 @@ sub users_can_hide { return 1; }
 
 sub default_show_name { 0 }
 
+sub pin_colour {
+    my ( $self, $p, $context ) = @_;
+    return 'green' if ($p->is_fixed || $p->is_closed) && $p->state ne 'not responsible';
+    return 'red' if $p->state eq 'unconfirmed' || $p->state eq 'confirmed';
```

Comment:
No unconfirmed report should be being shown on the site, so the first part of this line isn't needed.